tfjs-backend-webgpu/src/kernels/fill_webgpu.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =============================================================================
+ */
+import {util} from '@tensorflow/tfjs-core';
+import {computeDispatch, flatDispatchLayout} from '../webgpu_util';
+import {WebGPUProgram} from './webgpu_program';
+
+export class FillProgram implements WebGPUProgram {
+  variableNames: string[];
+  outputShape: number[] = [];
+  userCode: string;
+  dispatchLayout: {x: number[]};
+  dispatch: [number, number, number];
+  workPerThread = 4;
```

Comment:
Just curious, did you test other values for workPerThread and workGroupSize? Are these optimal for performance?

Reply:
I add performance data of five different workPerThread and workGroupSize (In the original comment). I didn't see any significant differences on macOS Mojave 10.14.4/Intel Iris Plus Graphics 655.